config/default.yml

```diff
@@ -153,6 +153,12 @@ Rails/Date:
     - strict
     - flexible
 
+Rails/DefaultScope:
+  Description: 'Avoid use of `default_scope`.'
+  Reference: 'https://andycroll.com/ruby/dont-use-default-scope/'
```

Comment:
I believe `Reference` is mostly used to refer to `edgeapi`, `wikipedia`.
Please add a `StyleGuide` link instead.